src/Propel/Generator/Model/Domain.php

```diff
@@ -190,11 +190,11 @@ public function getName(): ?string
     /**
      * Sets the domain name.
      *
-     * @param string $name
+     * @param string|null $name
      *
      * @return void
      */
-    public function setName(string $name): void
+    public function setName(?string $name): void
```

Comment:
I dont know if we need this, but the getter is nullable as well

> getName(): ?string

Reply:
I think it makes sense. If you can set it, you probably should be able to unset it